src/ArchitectNow.Web/Configuration/WebApiExtensions.cs

```diff
@@ -9,41 +9,40 @@
 
 namespace ArchitectNow.Web.Configuration
 {
-	public static class WebApiExtensions
-	{
-		public static void ConfigureApi(this IServiceCollection services, FluentValidationOptions fluentValidationOptions, Action<MvcOptions> configureMvc = null, Action<MvcJsonOptions> configureJson = null, Action<IMvcBuilder> configureMvcBuilder = null)
-		{
-			/*************************
+    public static class WebApiExtensions
+    {
+        public static void ConfigureApi(this IServiceCollection services, FluentValidationOptions fluentValidationOptions, Action<MvcOptions> configureMvc = null!, Action<MvcNewtonsoftJsonOptions> configureJson = null! , Action<IMvcBuilder> configureMvcBuilder = null!)
+        {
+            /*************************
              * IConfiguration is not available yet
              *************************/
 
-			services.AddAntiforgery(options => options.HeaderName = "X-XSRF-TOKEN");
-			services.AddRouting(options => options.LowercaseUrls = true);
-			var mvcBuilder = services.AddMvc(o =>

```

Comment:
just replaced `addMvc `with `AddControllers `and `addjsonOptions` with `AddNewtonsoftJson`. The rest is my editor running a formatting on the file